csm_web/frontend/static/frontend/css/style.css

```diff
@@ -207,9 +210,76 @@ header {
 	background-color: #007bff;
 	border-radius: 10px;
 	cursor: pointer;
-	left: 0;
-	right: 0;
-	bottom: 20px;
+}
+
+.affinity-btn {
```

Comment:
Since this type of button isn't necessarily just for affinity sections, I think it would be better to name it something more general like `manage-label-btn` or `label-btn` depending on how widely applicable you want this button style to be